forking.md

```diff
@@ -0,0 +1 @@
+https://github.com/mozhdeh-ahmadi/best_animals
```

Comment:
Your `birds.txt` and `fish.txt` files are still empty. Once you push some changes to the repo, I'll mark this as correct.